ext/hash/tests/gh12186_2.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Hash: bug #12186 - segfault when cloning a finalized context
+--FILE--
+<?php
+
+$c = hash_init('sha1');
+hash_final($c);
+clone $c;
```

Comment:
Wrap this in a try catch, same as the other test.

Also EOL before EOF nit

Reply:
Done.